pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -52,6 +70,8 @@ func (ds *dockerService) updateCreateConfig(
 		modifyHostNetworkOptionForContainer(false, podSandboxID, createConfig.HostConfig)
```

Comment:
For Hyper-V container, no need to call modifyHostNetworkOptionForContainer.

Reply:
Ack, will fix